DataCapturing.podspec

```diff
@@ -53,10 +55,15 @@ This framework can be included by your App if you are going to capture sensor da
   # So make sure tests are always located on the same level or below the podspec.
   s.test_spec 'Tests' do |test_spec|
     test_spec.source_files = 'DataCapturing/Tests/**/*.swift'
+    test_spec.resources = 'DataCapturing/Tests/**/*.sqlite'
   end
 
-  s.test_spec 'ExampleUnitTests' do |example_unit_test_spec|
-    example_unit_test_spec.source_files = 'ExampleUnitTests/**/*.swift'
+
+  s.app_spec 'Cyface' do |app_spec|
+    app_spec.source_files = 'Example/Source/**/*.swift'
+    app_spec.test_spec 'ExampleUnitTests' do |example_unit_test_spec|
+      example_unit_test_spec.source_files = 'ExampleUnitTests/**/*.swift'
+    end
```

Comment:
This change makes sure, that the tests for the Example app are able to find the App as a dependency, when running outside the IDE.